### DISCUSSION TITLE
Unable to authenticate

### DISCUSSION THREAD

**Discussion by Dongyifengs** [community/community#49518] [GitHub Education]
When I was uploading my school certificate (I made sure that my github information was perfect), I still gave me an error warning.

 - Your application cannot be reviewed until you fix the following:
You are significantly more likely to be verified if you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

**Comment by samus-aran** [comment]
Hi @Dongyifengs 👋 

Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍 Did you do anything differently in your next re-submission?

**Comment by Peins** [comment]
Me too,but I don't want to expose my real name to my followers.

**Comment by Gusarovv** [comment]
I found the following solution:
The problem is that this "error" also includes the fact that you are uploading an identity in a non-English language. Try to translate the ID and re-upload.
